### ISSUE TITLE
Persistent Volume Claims is Pending

### ISSUE THREAD

**Issue by kolesoffac** [hashicorp/consul-helm#48]
I install from helm.

`kubectl describe pvc`

Events:
  Type       Reason         Age              From                         Message
  ----       ------         ----             ----                         -------
  Normal     FailedBinding  7s (x2 over 7s)  persistentvolume-controller  no persistent volumes available for this claim and no storage class is set

In my cluster no persistent volumes and storage class.

Which pv and sc i need add to my cluster?

**Comment by krancour** [comment]
If this happens to be on minikube, then this could be a duplicate of #13, which also mentions a workaround.

**Comment by kolesoffac** [comment]
> If this happens to be on minikube, then this could be a duplicate of #13, which also mentions a workaround.

No, this to be on typical kubernetes cluster. It was install from scratch.

**Comment by krancour** [comment]
Sorry... I jumped the gun... the title of the PR looked so familiar to #13.

I'm looking more closely at your `kubectl describe pvc` and is says:

```
no persistent volumes available for this claim and no storage class is set
```

PersistentVolumeClaims must do one of three things:

1. Make a claim on an existing PV
2. Dynamically provision a new PV by making mention of the StorageClass that should be utilized in that process
3. Dynamically provision a new PV by relying on the cluster's default storage class.

By default, this chart utilizes situation no.3. But it would seem, perhaps, that your cluster does not actually _have_ a default StorageClass defined.

What do you get when you do the following?

```
$ kubectl get storageclasses
```

**Comment by kolesoffac** [comment]
```
NAME                      PROVISIONER                    AGE
local-storage (default)   kubernetes.io/no-provisioner   129m
```

I add this manually sc _local-storage_:
```
{
  "kind": "StorageClass",
  "apiVersion": "storage.k8s.io/v1",
  "metadata": {
    "name": "local-storage",
    "annotations": {
      "storageclass.kubernetes.io/is-default-class": "true"
    }
  },
  "provisioner": "kubernetes.io/no-provisioner",
  "reclaimPolicy": "Delete",
  "volumeBindingMode": "WaitForFirstConsumer"
}
```

**Comment by kolesoffac** [comment]
>> Do you have any better luck if you install / upgrade the chart with --set server.storageClass=local-storage?

Already try with same result pending.
